test/e2e/network_test.go

```diff
@@ -462,11 +464,13 @@ func (tc *testContext) getPodIP(selector metav1.LabelSelector) (string, error) {
 	return podList.Items[0].Status.PodIP, nil
 }
 
-// getWindowsServerContainerImage gets the appropriate WindowsServer image based on the cloud provider
+// getWindowsServerContainerImage gets the appropriate WindowsServer image based on the OS version
 func (tc *testContext) getWindowsServerContainerImage() string {
-	if tc.CloudProvider.GetType() == config.AWSPlatformType {
-		// On AWS we are currently testing 2019
+	switch tc.windowsServerVersion {
+	case windows.Server2019:
 		return "mcr.microsoft.com/powershell:lts-nanoserver-1809"
+	case windows.Server2022:
```

Comment:
Why is this a separate case if 2022 is the default?

Reply:
This is for clarity.